langlib/lang.array/src/main/ballerina/src/lang.array/array.bal

```diff
@@ -48,50 +66,173 @@ public function iterator(Type[] arr) returns abstract object {
     return arrIterator;
 }
 
+# Returns a new array comprising of position and member pairs.
+#
+# + arr - the array
+# + return - array of position, member pair
```

Comment:
```suggestion
# + return - the array of position and member pairs
```

Reply:
Should we use 'the' here as this array describing here does not exist, this function create and returns it?